integration/benchmark/follower.go

```diff
@@ -114,38 +150,21 @@ Loop:
 		default:
 		}
 
-		GetBlockByHeightTime := time.Now()
+		getBlockByHeightTime := time.Now()
 		block, err := f.client.GetBlockByHeight(f.ctx, f.height+1)
```

Comment:
f.height read needs to be guarded

Reply:
Done in https://github.com/onflow/flow-go/pull/3074, thanks!